tests/foreman/cli/test_import.py

```diff
@@ -273,6 +273,62 @@ def test_reimport_orgs_default_negative(self, test_data):
             'rm -rf ${{HOME}}/.transition_data {}'.format(files['users'])
         )
 
+    @data(*import_org_data())
+    def test_import_orgs_recovery(self, test_data):
+        """@test: Try to Import organizations with the same name to invoke
+        usage of a recovery strategy (rename, map, none)
+
+        @feature: Import Organizations Recover
+
+        @assert: 2nd Import will rename the new organizations, 2nd import will
+        map them and the 3rd one will resul in No Action Taken
+
+        """
+        # prepare the data
+        files = dict(self.default_dataset[1])
+        new_dataset = update_csv_values(files['users'], test_data)
+        ssh.upload_file(new_dataset, new_dataset)
+        files['users'] = new_dataset
+        # initial import
+        self.assertEqual(
+            Import.organization({'csv-file': files['users']}).return_code, 0)
+        # clear the .transition_data to clear the transition mapping
+        ssh.command('rm -rf ${HOME}/.transition_data')
+        # use the default (rename) strategy
+        self.assertIn(
```

Comment:
We usually do not perform verification against text messages, because it is a huge chance of failure on a 'non-English' environment. It is fine to verify return code or that stdout is not empty or that stderr is empty, but not text itself.
I propose to re-implement these verification (even if they look so nice). We can discuss that point more...